client/aliases.go

```diff
@@ -26,9 +26,12 @@ import (
 
 // aliasAction represents an action performed on aliases.
 type aliasAction struct {
-	Action  string   `json:"action"`
-	Snap    string   `json:"snap"`
-	Aliases []string `json:"aliases"`
+	Action string `json:"action"`
+	Snap   string `json:"snap"`
+	Target string `json:"target,omitempty"`
```

Comment:
the GET /aliases previous API was using "app"  for "snap.app"  , wondering if that would make this confusing? (this is really just the app without the snap part), I think the old API was probably wrong doing that